Patches/Moyo from the depth/Ammo/Ammo_Moyo.xml

```diff
@@ -320,6 +320,103 @@
 				</value>
 			</li>
 			
+			<!-- ==================== Abysteel Nail ========================== -->
+			
+			<li Class="PatchOperationAdd">
+				<xpath>Defs</xpath>
+				<value>
+					<!-- ==================== AmmoSets ========================== -->
+
+					<CombatExtended.AmmoSetDef>
+						<defName>AmmoSet_Abyssnail</defName>
+						<label>Moyo alloy nail</label>

```

Comment:
Why the swap to a nail in space of what used to be a somewhat vanilla charge ammo?